examples/ssh/linux_iplatency.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Domotz Custom Driver 
+ * Name: Linux Monitor IP Latency
+ * Description: This script is designed to ping an IP address and retrieve the average latency and packet loss percentage.
+ *   
+ * Communication protocol is SSH
+ * 
+ * Tested on Linux Distributions:
+ *      - Ubuntu 20.04 LTS
+ * Shell Version:
+ *      - Bash 5.1.16
+ * 
+ * Creates a Custom Driver table with a list of IP addresses, their average latency, and their packet loss.
+ * 
+ **/
+
+var pktno = "2"; // Number of packets to send during the ping command.
+var ipAddresses = ["8.8.8.8", "1.1.1.1", "192.168.0.1", "192.168.0.1", "172.17.0.1"]; // List of IP addresses to ping and retrieve status for.
+
+// Set up the SSH command options
+var sshCommandOptions = {
+    "timeout": 30000
+};
+
+var tableColumns = D.createTable(
+    "IP Latency",
+    [
+        { label: "IP Address" },
+        { label: "Latency", unit: "ms" },
+        { label: "Packet Loss", unit: "%" }
+    ]
+);
+
+// A function to check for authentication errors during execution
+function checkSshError(err) {
+    if (err.message) console.error(err.message);
+    if (err.code == 5) D.failure(D.errorType.AUTHENTICATION_ERROR);
+    console.error(err);
+    D.failure(D.errorType.GENERIC_ERROR);
+}
+
+function executeCommand(command){
+    var d = D.q.defer();
+    sshCommandOptions.command = command;
+    D.device.sendSSHCommand(sshCommandOptions, function (output, error) {
+        if(error) checkSshError(error);
+        d.resolve(output);
+    });
+    return d.promise;
+}
+
+/**
+ * Excuting a simple command to test access to device:
+ * 'ls' list the files and directories within a specified directory
```

Comment:
Do not use 'ls' for validation

Please properly check if "ping" command is supported on the host device with the needed parameters